src/main/java/org/opentripplanner/middleware/triptracker/TravelerLocator.java

```diff
@@ -136,10 +136,7 @@ public static TripInstruction alignTravelerToTrip(
         Locale locale = travelerPosition.locale;
 
         if (isApproachingEndOfLeg(travelerPosition)) {
```

Comment:
If this is at the start of a trip where the first leg is transit, there is no approaching the end of the leg.

Reply:
@binh-dam-ibigroup  If possible can you provide a trip which starts with a transit leg? I'm going to edit an existing trip (walk-to-bus-transition.json) to test with, but would prefer a real-world exampl